### ISSUE TITLE
Want to use vedo to display and reconstruct 3D CT images

### ISSUE THREAD

**Issue by hanjiahe1999** [marcomusy/vedo#1136]
I am an active beginner in vedo and would like to use vedo to display and reconstruct 3D CT images. 
I saw a demo on the official website about the head，
![20240606-234339](https://github.com/marcomusy/vedo/assets/128947985/c753ca43-b227-410f-bb79-ff87fb9da2b7)

I have a question about how to make a vti file like this. My image is a JPG.

Could you give me some advice, I would be grateful !!!

**Comment by marcomusy** [comment]
Hi, a JPG image is a 2D object. What you need instead is a 3D image (a `vedo.Volume()`).
A 3D image can be seen as a set of 2D slices one on top of the other.

**Comment by hanjiahe1999** [comment]
Yes, I have a set of CT slices (JPG images), which are converted into 3D images. Is there a special format for vti?

**Comment by hanjiahe1999** [comment]
I completed the rebuild but there is one small problem，When I zoom in, could I make the edges of the pixels disappear? 
![20240607-223312](https://github.com/marcomusy/vedo/assets/128947985/fbf6fdc2-7a51-48fc-ab5b-742bd9797de7)

**Comment by marcomusy** [comment]
Sorry I missed this issue! Have you tried `.linewidth(0)` ?
